fetching the integration version from the pyproject (#106)
port-labs/ocean: docs/framework-guides/docs/develop-an-integration/update-integration-code.md

## `pyproject.toml`

> This file consists of 3 main responsibilities: 

Reviewer: ```suggestion
This file is used for the following main goals:
```

> - `version` - the integration's current version, should be bumped when a new version of the integration is released

Reviewer: ```suggestion
- Maintain the integration's current version via the `version` field. Should be bumped when a new version of the integration is released
```

> - `dependencies` - the integration's dependencies, should be updated when new dependencies are added to the integration

Reviewer: ```suggestion
- Maintains the list of dependencies required by the integration, should be updated when new dependencies are added to the integration using the `poetry` CLI
```

> - Configuration for automated tools that ensure consistent code quality for every developed integration. These

Reviewer: ```suggestion
- Stores the configuration for automated tools that ensure consistent code quality for every developed integration. These
```
